### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Tresor11** [thoughtbot/upcase#2486]
I love using Upcase and want to make it as accessible as possible.

I'll use this issue to capture the work needed to make the website more accessible.

- [ ] Images must have alt text (9 Violations)
- Target: `.avatar__image`
- Target: `p:nth-child(2) > img -> <img src="/upcase/assets/marketing/many-faces-3bc8b3a405e3f84391e28460ff22c6623a07cc250bb12c1d2f78e0d772574ee8.png">`
- Target: `.conversation:nth-child(2) > .conversation__avatar`
- Target: `.conversation:nth-child(3) > .conversation__avatar`
- Target: `.card--two-per-line.card:nth-child(1) > .card__content > img`
- Target: `.card--two-per-line.card:nth-child(2) > .card__content > img`
- Target: `.page-content--reading.page-content > .media-object.testimonial__person > img`
- Target: `.page-section--split__column:nth-child(1) > .page-content--reading.page-content > .testimonial > .media-object.testimonial__person > img`
- Target: `.page-section--split__column:nth-child(2) > .page-content--reading.page-content > .testimonial > .media-object.testimonial__person > img`

- [ ] Links must have discernible text
- Target: `#my_account_link`
- Target: `a[href$="www.thoughtbot.com"]`

(I’m using the [aXe Chrome extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to find these errors)
